curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f8eeab71f4f24ba1a3a7eb.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 67f8eeab71f4f24ba1a3a7eb
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+After the form, add a `div` element with an `id` of `results` that will display the search results.
```

Comment:
If you ask to put it after the form, then you should check that it comes after it.